src/components/Comments/Comment.js

```diff
@@ -170,6 +244,13 @@ const Comment = ({
             </ButtonHightlight>
           </TouchableOpacity>
         )}
+        <ButtonHightlight
+          onLongPress={handleOnLongPress}
+          style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+          onPress={() => refSheet.current.open()}>
+          <MemoSendDM width={20} height={18} />
+        </ButtonHightlight>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new button is added to open the bottom sheet menu when pressed. The SVG icon `MemoSendDM` is used for this button. No issues found here.
```diff
+ <ButtonHightlight
+   onLongPress={handleOnLongPress}
+   style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
+   onPress={() => refSheet.current.open()}>
+   <MemoSendDM width={20} height={18} />
+ </ButtonHightlight>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->